examples/configs/sensors/FLIR_rgb_1.yml

```diff
@@ -1,7 +1,7 @@
 sensor_id: FLIR_rgb_1
-temporal_res: 195.0
-spatial_res_x: 548.0
-spatial_res_y: 800.0
+fps: 195.0
+sizex: 548.0
+sizey: 800.0
```

Comment:
width_px and height_px may be more intuitive, but fine with this for now.

Reply:
IIRC, it's what the call to the sensor library wrapper was using. I was trying to eliminate renames between the yaml file and the last bit of code. Also, I didn't feel I had the knowledge to second guess what we had there.

Reply:
I think we should have consistent names throughout our code, and rename as necessary for the libraries at the "last step". Since we have a lot of cameras, we should use consistent terminology for image size.